frontend/src/components/BookTable/index.tsx

```diff
@@ -369,30 +375,33 @@ const BookTable = ({
     };
   }, []);
 
-  const paymentObj = useCallback((width: number) => {
-    return {
-      field: 'payment_method',
-      headerName: fav.mode === 'fiat' ? t('Payment Method') : t('Destination'),
-      width: width * fontSize,
-      renderCell: (params: any) => {
-        return (
-          <div
-            style={{ cursor: 'pointer' }}
-            onClick={() => {
-              onOrderClicked(params.row.id, params.row.coordinatorShortAlias);
-            }}
-          >
-            <PaymentStringAsIcons
-              othersText={t('Others')}
-              verbose={true}
-              size={1.7 * fontSize}
-              text={params.row.payment_method}
-            />
-          </div>
-        );
-      },
-    };
-  }, [fav.mode]);
+  const paymentObj = useCallback(
+    (width: number) => {
+      return {
+        field: 'payment_method',
+        headerName: fav.mode === 'fiat' ? t('Payment Method') : t('Destination'),
+        width: width * fontSize,
+        renderCell: (params: any) => {
+          return (
+            <div
+              style={{ cursor: 'pointer' }}
+              onClick={() => {
+                onOrderClicked(params.row.id, params.row.coordinatorShortAlias);
+              }}
+            >
+              <PaymentStringAsIcons
+                othersText={t('Others')}
+                verbose={true}
+                size={1.7 * fontSize}
+                text={params.row.payment_method}
+              />
+            </div>
+          );
+        },
+      };
+    },
+    [fav.mode],
+  );
```

Comment:
I think all modifications on this file was caused by the lintern, I'm pretty sure I didn't change anything here